3-node-cluster/pgpool-internals/follow_primary.sh

```diff
@@ -45,7 +45,7 @@ fi
 ssh postgres@$NEW_MAIN_NODE_HOST "psql -d postgres -w -c \"SELECT pg_create_physical_replication_slot('slot_$SLOT_NAME');\""
 
 # Update connection string on all standby nodes to point to new primary
-ssh postgres@$NODE_HOST "psql -d postgres -w -c \"ALTER SYSTEM SET primary_conninfo = 'user=repuser host=$NEW_MAIN_NODE_HOST port=$NODE_PORT';\""
+ssh postgres@$NODE_HOST "PGPASSFILE=~/.pgpass psql -d postgres -w --set=sslmode=disable -c \"ALTER SYSTEM SET primary_conninfo = 'user=repuser host=$NEW_MAIN_NODE_HOST port=$NODE_PORT';\""
```

Comment:
I have some confusion about --set=sslmode=disable is syntax correct and if yes why we are disabling ssl-mode?

Reply:
You are right this was not in out initial code, it must have been added mistakenly while editing this, I'll remove it